src/random_bq_to_anndata.py

```diff
@@ -0,0 +1,46 @@
+from google.cloud import bigquery
+import argparse
+import random
+import anndata as ad
+
+
+def get_random_ids(project, dataset, num_cells):
+    client = bigquery.Client()
+    query = client.query(f"SELECT MAX(cas_cell_index) AS max_table_number FROM `{project}.{dataset}.cas_cell_info`")
+    max_cell_id = int([row.max_table_number for row in list(query.result())][0])
+    print(f"Getting {num_cells} random IDs between 0 and {max_cell_id}...")
+    cell_ids = list(range(0, max_cell_id))
```

Comment:
`range` excludes its upper limit so
```suggestion
    cell_ids = list(range(0, max_cell_id + 1))
```